packages/core/src/components/Form.js

```diff
@@ -428,7 +428,7 @@ if (process.env.NODE_ENV !== "production") {
     onSubmit: PropTypes.func,
     id: PropTypes.string,
     className: PropTypes.string,
-    tagName: PropTypes.string,
+    tagName: PropTypes.oneOfType([PropTypes.string, PropTypes.elementType]),
```

Comment:
I think I should be able to make this just `PropTypes.elementType` 🤔

Reply:
Hmm, does PropTypes.elementType include strings?

Reply:
Yeah, it's a `ReactComponentLike`:
```ts
export type ReactComponentLike =
    | string
    | ((props: any, context?: any) => any)
    | (new (props: any, context?: any) => any);
```